### DISCUSSION TITLE
Can you edit your installation guide?

### DISCUSSION THREAD

**Discussion by salaheddinelhamraoui** [daisyui/react-daisyui#155] [General]
Hello there,

Thanks for this package, I like using it with React.

The first time I installed the package I struggled to setting it up correctly, I followed the documentation you provided on Npm website but didn't succeed to make it work successfully.

The problem was that I need to import this file "node_modules/daisyui/dist/**/*.js" in tailwind.config.js.

I think it will be helpful for other developers if you add this step to your installation guide.

**Comment by imtheaman** [comment]
Hey,
you can simply pass require('daisyui') in your plugins array inside tailwind.config.js.
```
module.exports = {
  //...
  plugins: [require("daisyui")],
}
```

**Reply by salaheddinelhamraoui** [reply]
I did that but still didn't work, till I imported it inside content

```
module.exports = {
  content: ["./src/**/*.{js,jsx,ts,tsx}", "node_modules/daisyui/dist/**/*.js"],
  theme: {
    extend: {},
  },
  plugins: [require("daisyui")],
  daisyui: {
    themes: ["light", "cmyk"],
  },
};
```
